_data/domains.yml

```diff
@@ -201,6 +201,12 @@ websites:
       tfa: No
       twitter: HeartInternet
 
+    - name: Hexonet
+      url: http://www.hexonet.net
+      tfa: Yes
+      software: Yes
+      doc: https://cp.hexonet.net/cp2/downloads/2FA.pdf
```

Comment:
Do you know if they have the documentation available on a normal webpage - not a PDF file? If so then please use that instead.